src/index.js

```diff
@@ -123,26 +123,119 @@ export default class ToggleBlock {
     this.wrapper.classList.add('toggle-block__selector');
     this.wrapper.id = crypto.randomUUID();
 
     const icon = document.createElement('span');
+    const input = document.createElement('div');
+    const defaultContent = document.createElement('div');
 
     icon.classList.add('toggle-block__icon');
     icon.innerHTML = this.data.status === 'closed' ? toggleIconClosed : toggleIconOpen;
 
-    const input = document.createElement('div');
-
     input.classList.add('toggle-block__input');
     input.contentEditable = true;
-    input.addEventListener('keyup', this.createParagraphFromToggleRoot.bind(this));
     input.innerHTML = this.data.text || '';
 
+    // Events to create other blocks and destroy the toggle
+    input.addEventListener('keyup', this.createParagraphFromToggleRoot.bind(this));
+    input.addEventListener('keydown', this.removeToggle.bind(this));
+
+    // Establishes the placeholder for the toggle root when it's empty
+    input.addEventListener('keyup', this.setPlaceHolder.bind(this));
+    input.setAttribute('placeholder', 'Toggle');
+
+    // Calculates the number of toggle items
+    input.addEventListener('focus', this.calculateChildren.bind(this));
+    input.addEventListener('focusout', this.calculateChildren.bind(this));
+    input.addEventListener('focus', this.setDefaultContent.bind(this));
+    input.addEventListener('focusout', this.setDefaultContent.bind(this));
+
+    defaultContent.classList.add('toggle-block__content-default');
+    defaultContent.setAttribute('hidden', true);
+    defaultContent.innerHTML = 'Empty toggle. Click or drop blocks inside.';
+    defaultContent.addEventListener('click', this.clickInDefaultContent.bind(this));
+
     this.wrapper.appendChild(icon);
     this.wrapper.appendChild(input);
+    this.wrapper.appendChild(defaultContent);
+  }
+
+  /**
+   * Adds the actions to do when the default content is clicked.
+   */
+  clickInDefaultContent() {
+    const originalIndex = this.api.blocks.getCurrentBlockIndex();
+    const foreignKey = this.wrapper.id;
+    const index = originalIndex + 1;
+    const id = crypto.randomUUID();
+
+    this.api.blocks.insert();
+
+    const newBlock = this.api.blocks.getBlockByIndex(index);
+
+    newBlock.holder.firstChild.firstChild.classList.add('toggle-block__item');
+    newBlock.holder.setAttribute('foreignKey', foreignKey);
+    newBlock.holder.setAttribute('id', id);
+
+    document.getElementById(id).firstChild.firstChild.focus();
+    this.setDefaultContent();
+  }
+
+  /**
+   * Sets the default content. If the toggle has no other blocks inside it,
+   * so remove the hidden tag in the default content, otherwise adds the hidden tag.
+   */
+  setDefaultContent() {
+    const children = document.querySelectorAll(`div[foreignKey="${this.wrapper.id}"]`);
+
+    if (children.length === 0) {
+      this.wrapper.lastChild.removeAttribute('hidden');
+    } else {
+      this.wrapper.lastChild.setAttribute('hidden', true);
+    }
```

Comment:
@juliansg022 what about using object destructing to simplify?

```js
const { lastChild } = this.wrapper;
if children.length === 0 ? lastChild.removeAttribute('hidden') : lastChild.setAttribute('hidden', true);
```

Reply:
@juanhurtado10 You're right, it's better, but I'll keep the if-statement because if I use the ternary operator in this case, the linter shows me a warning.